### PR TITLE
Added the label with (click Ok to export data) to dlgExportDataset

### LINKED ISSUE #5530: Export datasets dialog
image
image

Is there any reason why we do not have a similar text perhaps(**click Ok to export data)** to notify the user to click ok to export data? Perhaps it could be bolded or change the colour of the text to alert users? Users tend to forget to click Ok thinking that they have saved/exported data.

### PR DESCRIPTION
**Pull request by Wycklife**
Fixes #5530
Fixes the last part of #4347
@africanmathsinitiative/developers Someone to review this.

### CODE REVIEW

**Comment by Wycklife** [comment]
@africanmathsinitiative/developers The design seems not okay from my end. Someone to kindly redesign it and then merge the changes back to this branch.

File: instat/dlgExportDataset.vb [hunk 1]
```diff
@@ -43,6 +43,8 @@ Public Class dlgExportDataset
         ucrAvailableSheets.SetText("Data Frame to Export:")
 
         ucrInputExportFile.SetParameter(New RParameter("file", 1))
+
+        lblClicktoConfirm.Text = "click Ok to export data"
```
Comment (maxwellfundi) [inline instat/dlgExportDataset.vb]: Can we have this in a different color just to make sure the users attention is attracted to it?  what do you think @rdstern 
 @maxwellfundi  I quick small one, the case for the label text would be better if it read - "Click OK to export data"

Looks good though!

Comment (maxwellfundi) [reply instat/dlgExportDataset.vb]: @Wycklife  would you still want the designer checked on this?

Comment (Wycklife) [reply instat/dlgExportDataset.vb]: @maxwellfundi Yes someone to look at the designer.

Comment (Wycklife) [reply instat/dlgExportDataset.vb]: @maxwellfundi I have just changed the label text as suggested by Roger, should I change the color of the label text?

**Comment by Wycklife** [comment]
@rdstern This contain both changes from the Export and calculator dialogs.

**Comment by rdstern** [comment]
ok thanks. I will have a look.

**Comment by rdstern** [comment]
@Wycklife that is just the change I was hoping for in the keyboard.
1. Sorry to be "picky", but could you add a <space> between the name of the key and the type, e.g. coin (f) rather than coin(f).
2. Shorten the long texts on: 
internet_browser > browser
wakefield_month > month
wakefield_upper > upper
wakefield_year > year
3. On the 3 wakefield keys, edit your code, so the wakefield_month becomes just wakefield::month() etc.  Currently the code on each of these 3 keys gives an error.
4. When I return to the dialogue it sometimes cannot show the bottom line.  Could the width of the keys be very slightly reduced?
5 Hair(f)  > hair (f)  - no capital H
6. Still some types to add, that I missed before:
pet > pet (f)
primary > primary (f)
died > died (L)
dna > dna (f)
internet_browser > browser (f)
marital > marital (f)
gender > gender (f)

Then I will also work on providing suggested tooltips for them.

**Comment by maxwellfundi** [comment]
@Wycklife  Conflicts again here.

**Comment by Wycklife** [comment]
The changes on the calculator has been done on a new pull request. Now I'm closing this due to conflict

**Pull request closed without merging**